samples/shields/x_nucleo_iks01a3/sensorhub/src/main.c

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <sensor.h>
+#include <stdio.h>
+#include <misc/util.h>
```

Comment:
Should be `sys/utils.h` according to on recent: https://github.com/zephyrproject-rtos/zephyr/pull/16540

Reply:
ok, it makes sense.

Reply:
Done. I did it for lps22hh and lsm6dso yesterday. But I forgot to apply the same here.
Thx!